res/css/views/elements/_EventTilePreview.scss

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2016 OpenMarket Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.mx_EventTilePreview_loader {
+    &.mx_IRCLayout,
+    &.mx_GroupLayout {
```

Comment:
This doesn't seem neccessary?

Reply:
mx_GroupLayout has it's own padding so I want to be more explicit to override it.

Reply:
Ah :+1: